internal/api/config/config.go

```diff
@@ -19,7 +19,7 @@ type StandardConfig struct {
 
 type ServerConfig struct {
 	StandardConfig
-	LocalMode                   bool
+	LocalMode                   bool // this will never be set from the environment
```

Comment:
I think a little more detail here may help.

```suggestion
	// LocalMode is true if the server is running as a non-containerized process
	// that was launched via `kargo server`, which uses the kargo CLI user's
	// current kubeconfig for all cluster interactions.
	LocalMode                   bool // this will never be set from the environment
```